src/api/libcellml/model.h

```diff
@@ -72,7 +72,7 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
      *
      * @param units The units to add.
      */
-    void addUnits(const UnitsPtr &units);
+    bool addUnits(const UnitsPtr &units);
```

Comment:
Missing documentation for return statement.

Reply:
Done in 249c836e.